src/main/java/seedu/algobase/model/AlgoBase.java

```diff
@@ -157,10 +154,19 @@ public boolean checkIsProblemUsed(Problem problem) {
      * Removes the given problem from all plans
      * @param problem the problem to be removed
      */
-    public void removeProblemFromAllPlans(Problem problem) {
+    void removeProblemFromAllPlans(Problem problem) {
```

Comment:
Making these methods package private might not be necessary, can you justify?

Reply:
They are not needed outside model package, so no need to expose them as public.